auth/realm/base/src/main/java/org/wildfly/security/auth/realm/AggregateSecurityRealm.java

```diff
@@ -78,12 +78,18 @@ public AggregateSecurityRealm(final SecurityRealm authenticationRealm, Function<
     public RealmIdentity getRealmIdentity(final Evidence evidence) throws RealmUnavailableException {
         boolean ok = false;
         final RealmIdentity authenticationIdentity = authenticationRealm.getRealmIdentity(evidence);
+        if (authenticationIdentity.exists()) {
+            ElytronMessages.log.authenticationIdentityFound(evidence.getDecodedPrincipal());
```

Comment:
@lvydra Thanks for working on this! To be consistent with other realms, I think it would be good to use trace logging in this file instead, e.g., see https://github.com/wildfly-security/wildfly-elytron/blob/1.x/auth/realm/ldap/src/main/java/org/wildfly/security/auth/realm/ldap/LdapSecurityRealm.java#L662.